Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'dotenv-rails'
 gem 'devise'
 gem 'database_cleaner'
-
+gem 'carrierwave'
```

Comment:
Remove this gem and add it in the PR where you actually get file uploading working